pages/AdaptivePostList/index.js

```diff
@@ -6,28 +6,13 @@ import AnimationPostList from '../../components/AnimationPostList';
 import SimplePostList from '../../components/SimplePostList';
 import Nav from '../../components/Nav';
 import posts from '../../data/posts';
-import { useMemoryStatus } from '../../utils/hooks';
 
 const Loading = () => <Fragment>Loading...</Fragment>;
 
 const AdaptivePostList = () => {
-  const memoryStatus = useMemoryStatus();
-  const {
-    manualEnabled,
-    isAnimationOn
-  } = useContext(AnimationEmulationContext);
-  
-  if (!memoryStatus) return <Loading />;
-  const { overLoaded } = memoryStatus;
+  const { animationEnabled } = useContext(AnimationEmulationContext);
```

Comment:
let's rename this to `animationAllowed`.

Reply:
renamed.